.github/workflows/macos-bazel.yml

```diff
@@ -14,7 +14,7 @@ permissions:
 jobs:
   bazel:
     name: bazel + ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}-large
+    runs-on: macos-latest
```

Comment:
`*-latest` can break the builds in unexpected ways, i.e., when GHA changes what specific version `latest` points to.  Better to use `macos-14` and then upgrade when ready to `macos-15` or whatever comes next.